notebook/tests/selenium/utils.py

```diff
@@ -195,13 +195,22 @@ def wait_for_stale_cell(self, cell):
         wait = WebDriverWait(self.browser, 10)
         element = wait.until(EC.staleness_of(cell))
 
+    def wait_for_element_availability(self, element):
+        _wait_for(self.browser, By.CLASS_NAME, element, visible=True)
+
     def get_cells_contents(self):
         JS = 'return Jupyter.notebook.get_cells().map(function(c) {return c.get_text();})'
         return self.browser.execute_script(JS)
 
     def get_cell_contents(self, index=0, selector='div .CodeMirror-code'):
         return self.cells[index].find_element_by_css_selector(selector).text
 
+    def get_cell_output(self, index=0, output='output_subarea'):
+        try:
+            return self.cells[index].find_element_by_class_name(output).text
```

Comment:
This is singular, so it's only going to be the first output of the cell, which may have multiple outputs. Maybe we should call it `get_cell_outputs`, and return whatever output elements there are.

Reply:
@takluyver sorry, I don't understand the multiple outputs part. Do you mean for an output that has several lines? Like using a loop that prints the value line by line? The point of the method is to get the whole text on the output area of a single cell. So, if we put a loop on the cell, the output, even if there are several lines, will be captured, the detail would be that the result would be a string with `\n` as breaklines. If someone would like to work on that output separately, a split method would be required. 

Having said that, I may have misunderstood your perspective 😅, so tell me if I'm far from what you commented lol

Reply:
Conceptually, in a notebook, each cell can have zero-to-many outputs, each of which may be one of various different types, e.g. plain text, HTML, or an image. See here for more info: https://nbformat.readthedocs.io/en/latest/format_description.html#code-cell-outputs

So I'm suggesting that rather than returning text, this method should return a list of browser elements for the outputs from that cell. That also conveniently means we don't need a special case `None` for when there's no output - it can just return an empty list then.